test6/src/Main.java

```diff
@@ -0,0 +1,32 @@
+package com.company;
+import java.io.*;
+import java.util.*;
+
+public class Main {
+    private static final Scanner scanner = new Scanner(System.in);
+    private static Solution solution = new Solution();
+
+    public static void main(String[] args) throws IOException {
+        BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter("out.txt")); //System.getenv("OUTPUT_PATH")
+
+        int n = scanner.nextInt();
+        scanner.skip("(\r\n|[\n\r\u2028\u2029\u0085])?");
+
+        String s = scanner.nextLine();
+
+        solution.setN(n);
+        solution.setS(s);
+
+        int result = solution.func1(solution.getN(), solution.getS());
```

Comment:
зачем передавать в метод  что-то? Объект уже все имеет.
```suggestion
        int result = solution.func1(solution.getN(), solution.getS());
```